tests/test_multilabel_classification.py

```diff
@@ -271,3 +283,19 @@ def test_get_cross_validated_multilabel_pred_probs(dummy_features, labels, cv, p
     # Gold master test - Ensure output is consistent
     assert dummy_features.shape == (10, 2)
     assert np.allclose(pred_probs, pred_probs_gold, atol=5e-4)
+
```

Comment:
Is the complaint by codecov: https://github.com/cleanlab/cleanlab/runs/9110581266
Something that can be addressed? If not, any idea why  the coverage decreased so drastically?

Reply:
it passes in Elias's branch, so wondering why it's not covered in mine, I can parametrize alpha

Reply:
It may be you've deleted more (previously covered) code from a file or something like that which confuses codecov to thinking the percentage of covered lines as dropped

Reply:
yep 👍

Reply:
Fixed: https://github.com/aditya1503/cleanlab/blob/26d79fce8efb444f8c27b6a391c7e3fafff18202/tests/test_multilabel_classification.py#L288